### PR TITLE
fix: ENTESB-10521 Not visible items in Fuse Console Preferences

### PR DESCRIPTION
**Pull request by alexkieling**

### CODE REVIEW

File: packages/online/src/main.less [hunk 1]
```diff
@@ -6,7 +6,6 @@
 @pf-padding: @grid-gutter-width / 2;
 
 #main {
```
Comment (astefanutti) [inline packages/online/src/main.less]: Out of curiosity, where is the `#main` selector declared in the recent versions?

**Comment by alexkieling** [comment]
I've tested the master and 1.x-redhat-7.x branches and they seem fine. I think this issue is only happening in versions up to 7.3.

File: packages/online/src/main.less (hunk 1 above)
Comment (alexkieling) [reply packages/online/src/main.less]: It's declared here: https://github.com/hawtio/hawtio-core/blob/4d91ea331156528df1342f12b935c4ca4e32de3d/src/app/page/page.component.ts#L89

Comment (astefanutti) [reply packages/online/src/main.less]: Is this really matching the `#main` selector? I would have expected it should use a CSS attribute selector, like `[role=main]`.

**Comment by astefanutti** [comment]
OK, that must explain why I could not reproduce on newer versions.

File: packages/online/src/main.less (hunk 1 above)
Comment (alexkieling) [reply packages/online/src/main.less]: Good point. I think recent versions don't set the "main" id anymore.

Comment (astefanutti) [reply packages/online/src/main.less]: OK that was my understanding. Thanks for the confirmation.

**Pull request merged**